.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,40 @@
+## Description
+
+
+- Fixes # (issue) -> A reference to a related issue in your repository
+
+- A clear, concise description of the changes proposed in the pull request
+
+- @mentions of the person or team responsible for reviewing proposed changes (Optional)
+
+## Type of change
+
+Describe the change you're making: how it affects the API, user experience...
```

Comment:
this type of change also go in the tags? Or not?

Reply:
Could be useful to have it on the tags as well :)